src/hud.h

```diff
@@ -175,7 +175,11 @@ class Hud {
 	v3f m_selected_face_normal;
 
 	video::SMaterial m_selection_material;
-	bool m_use_selection_mesh;
+	
+	enum { 
```

Comment:
enum Mode {}; creates a non-anonymous enum and that, I believe, is not needed nor required here.